doc/user/layouts/shortcodes/cloud-connection-details.html

```diff
@@ -1,8 +1,20 @@
-1. Click the deployment card, then **download** and **unzip** the certificate package.
+1. Click on a deployment to see the individual deployment view.
+
+1. If you are connecting through Tailscale VPN, go to the Integrations card, then the Tailscale tab. Click **Enable**, then enter the Tailscale auth key in the Tailscale Authentication dialog and click **Save**.
+
+    For help generating an auth key, please see the [Tailscale documentation](https://tailscale.com/kb/1085/auth-keys/).
```

Comment:
We probably want to tell them that the key type should be a one-off key.

Reply:
Added.